sed/loader/base/loader.py

```diff
@@ -50,5 +50,38 @@ def read_dataframe(
         and a dictionary with metadata"""
         return None, None
 
+    @abstractmethod
+    def get_count_rate(
+        self,
+        fids: Sequence[int] = None,
+        **kwds,
+    ) -> Tuple[np.ndarray, np.ndarray]:
+        """
+        Create count rate data for the files specified in ``fids``.
```

Comment:
Perhaps a generailzed implementation can already be in the base loader and the further additions in the plugins?
I also see that in generic loader, it is just returing None.

Reply:
I can't quite comment on many of other additions but I checked them and looked fine.

I would really push the idea of some documentation, because the codebase ported from mpes is getting quite complex and would really benefit from just at least having the docstrings in another file. The documentation is already setup so should be easy to add further modules. I, otherwise, do plan to work on it #99 (after steinn's update). Let me know if you can't figure it out for now.

Reply:
An abstract class (in our case, the base loader) is indeed a template, but can also contain implementations. Even if it's implemented, it needs to be overloaded by the subclasses. But the benefit is it just reduces the same code occuring multiple times. The subclass can just call super().my_abstract_method() in it's implementation and that's a valid use case. 

But this comes down to personal preference then. I prefer when the abstract classes already contain a general implementation, and the subclasses just overriding/adding to those implementations.

Reply:
In principle I agree, however in this case I don't really know what to implement in the abstract class. You can make some suggestions in a new PR yourself.

Reply:
When looking into this, I realized that fixing the autodoc will require some more work, so I will even do only this in a separate PR.

Reply:
I created the PR #106 with updated docstrings and working documentation.